data/172/939/590/1/1729395901.geojson

```diff
@@ -5,11 +5,17 @@
     "edtf:cessation":"uuuu",
     "edtf:inception":"uuuu",
     "geom:area":0.018564,
-    "geom:area_square_m":165354534.501977,
+    "geom:area_square_m":165354534.501948,
     "geom:bbox":"-101.064385,43.810083,-100.944638,43.995082",
     "geom:latitude":43.91662,
     "geom:longitude":-101.002007,
     "iso:country":"US",
+    "label:eng_x_preferred_placetype":[
+        "UT"
```

Comment:
Hmm some of these values look incorrect, and more like abbreviations. I'd expect this to be "township" or similar..

Reply:
Yeah, that `UT` abbreviation isn't satisfying is it?

In most cases when I also `qs:type` value available I'd expand that to `township` or `town` or `city` & etc.

In this case I only have the following Census properties:

```
    "uscensus:countyfp":75,
    "uscensus:geoid":4607525500,
    "uscensus:name":"Grandview",
    "uscensus:namelsad":"Grandview UT",
    "uscensus:statefp":46,
```

Reviewing the US labels:

```
Precinct
township
purchase
city
reservation
Reservation
district
town
gore
charter township
precinct
plantation
borough
village
UT
grant
municipality
location
```

Ignoring Census' weird practice of upper/lower initial casing some off those, `UT` is the only one that stands out as still being abbreviated, and there are 290 records (a 0.1% problem).

Web search is tricky since UT also means Utah the state... but looking at example GeoIDs I'm fairly confident this stands for `unorganized territory` (like in [Maine](https://en.wikipedia.org/wiki/List_of_unorganized_territories_in_Maine), South Dakota, North Dakota, Minnesota, and one record in Indiana.)

So I'll edit just those 286 records to spell out UT as `unorganized territory` and 4 to spell out `township`.

I also 18 `Precinct` records that have garbled labels from a bad substr() so I'll fix those, too.

Reply:
Fixed in https://github.com/whosonfirst-data/whosonfirst-data-admin-us/pull/156/commits/32550f7ae4b1c33b97efdba26eead848ec54cca8.